.github/workflows/release.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           registry-url: "https://npm.pkg.github.com"
-      - name: scope package name as required by GHPR
+      - name: scope package name as required by GHCR
```

Comment:
?

Reply:
Maybe we should not use an acronym here. While ghcr seems to encompass npm, its core business is docker.

Reply:
Yes. GitHub mostly seems to talk about `GitHub Packages` in https://docs.github.com/en/packages

There's one user who they quote saying `GPR`,
and only two doc pages even yield search results for it, e.g. https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net
and one for Ruby.
Google hits for GPR (in Canada?) generally are for `Ground Penetrating Radar` (and if you don't know why, you don't want to know).